src/main/java/network/brightspots/rcv/CastVoteRecord.java

```diff
@@ -149,7 +149,7 @@ void logRoundOutcome(
       logStringBuilder.append(" [value] ").append(fractionalTransferValue);
     }
 
-    Logger.fine(logStringBuilder.toString());
+    Logger.info(logStringBuilder.toString());
```

Comment:
I had to revert this because it made the console even busier than it already is, even causing RCTab to crash when running `2013_minneapolis_mayor_scale`.

Reply:
Good catch, that was unintentional